docs/index.asciidoc

```diff
@@ -33,6 +33,26 @@ type of event can be enriched and transformed with a broad array of input, filte
 native codecs further simplifying the ingestion process. Logstash accelerates your insights by harnessing a greater
 volume and variety of data.
 
+
+[serverless]
+.Logstash to {serverless-full}
+****
+You'll use the {ls} <<plugins-outputs-elasticsearch,{es} output plugin>> to send data to {serverless-full}.
+Note these differences between {es-serverless} and both {ess} and self-managed {es}:
+
+* Use *API keys* to access {serverless-full} from {ls}. 
+Any user-based security settings in your in your <<plugins-outputs-elasticsearch,{es} output plugin>> configuration are ignored and may cause errors.
+* {serverless-full} uses *{dlm} ({dlm-init})* instead of {ilm} ({ilm-init}). 
```

Comment:
Could we add a link to DLM documentation?

Reply:
Nice suggestion. DONE!